amivapi/users/security.py

```diff
@@ -78,31 +79,55 @@ def create_user_lookup_filter(self, user_id):
             return None
 
 
-def hide_fields(response):
+@on_post_hook
+def hide_after_request(request, response, payload):
+    """Hide user fields after all requests to /users.
+
+    Works with GET, POST as well as PATCH.
+
+    Args:
+        request, response: unused
+        payload (dict): response data
+    """
+    # Use either the '_items' field (resource requests) or the
+    # whole payload as one item (item requests)
+    for item in payload.get('_items', [payload]):
+        _hide_fields(item)
+
+
+def hide_after_fetch(item):
```

Comment:
Do we need this function? We could just use _hide_fields directly instead.
Or if we want the extra name we could do:

```
hider_after_fetch = _hide_fields
```